handler.go

```diff
@@ -91,6 +92,33 @@ func (h *GODNSHandler) do(Net string, w dns.ResponseWriter, req *dns.Msg) {
 	}
 	logger.Info("%s lookup %s", remote, Q.String())
 
+	// We don't care of other type of requests than A, so fail fast returning an
+	// empty DNS response, NOERROR status and the SOA in the Authority Section as
+	// required by RFC https://datatracker.ietf.org/doc/html/rfc2308
+	if len(settings.Hosts.Zone) > 0 && q.Qtype != dns.TypeA && strings.HasSuffix(q.Name, settings.Hosts.Zone) {
```

Comment:
Should we send a servfail if the query name doens't have the Zone suffix ?

Reply:
Instead of forwarding then?

Reply:
Yeah

Reply:
To say: "godns is for 1 zone only"?

Reply:
Exactly